source/form.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Подбор программы для кота и кошек</title>
+</head>
+<body>
+<header class="page-header">
+  <div class="page-header__div">
+    <a class="cat-logo" href="index.html"><img class="cat-logo__image" src="#" alt="Логотип компании Кэт Энерджи."></a>
+    <a class="logo" href="index.html"><img class="logo__image" src="#" alt="Текстовый логотип Кэт Энерджи."></a>
+    <button class="page-header__button-close button-close" type="button" aria-label="Закрыть"></button>
+  </div>
+  <nav class="site-navigation">
+    <ul class="site-navigation__list site-nav-list">
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="catalog.html">Каталог продукции</a></li>
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="index.html">Главная</a></li>
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="form.html">Подбор программы</a></li>
+    </ul>
+  </nav>
+</header>
+<main class="page-main">
+  <h1 class="page-main__headline headline">Подбор программы</h1>
+  <p class="page-main__request">Заполните анкету и мы подберем программу питания для вашего кота</p>
+  <form class="page-main__form form" action="https://echo.htmlacademy.ru" method="post">
+    <fieldset class="form__fieldset cat-info-fieldset">
+      <ul class="form__list cat-info-list">
+        <li class="cat-info-list__item">
+          <label class="form__cat-info-label" for="cat-name">Имя:*</label>
+          <input class="form__cat-info-input" id="cat-name" name="cat-name" placeholder="Барсик" type="text" required>
```

Comment:
```suggestion
          <input class="form__input-text" id="cat-name" name="cat-name" placeholder="Барсик" type="text" required>
```
Хотя можно просто input, тк умолчательно подразумевается что он текстовый